sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -226,7 +236,10 @@ object ScalaNativePluginInternal {
             val isCppSource = path.endsWith(".cpp")
 
             val compiler = abs(if (isCppSource) clangpp else clang)
-            val flags    = (if (isCppSource) Seq("-std=c++11") else Seq()) ++ opts
+            val flags = (if (isCppSource)
+                           (if (isWindows) Seq("-std=c++14")
```

Comment:
it suddenly includes <algorithm> file in the long chain of includes which has 'auto' as return method, so:
"\include\algorithm:2522:2: error: 'auto' return without trailing return type; deduced return types
 are a C++14 extension"